src/services/supabase/products/product.utils.ts

```
@@ -0,0 +1,86 @@
+
+    // Sort products
+    return [...filtered].sort((a, b) => {
+        switch (sortBy) {
+            case 'price_asc':
+                return a.price - b.price;
+            case 'price_desc':
+                return b.price - a.price;
+            case 'newest':
+            default:
+                // Sort by product_id as a proxy for date
+                const idA = a.product_id;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)